test_assets/foundation/stage_configuration/metersPerUnit/README.md

```diff
@@ -0,0 +1,34 @@
+# MetersPerUnit
+
+These files show how the `metersPerUnit` stage metadata works.
+
+## MetersPerUnit = 1
+
+[metersPerUnit_1.usda](./metersPerUnit_1.usda)
+
+```usda
+metersPerUnit = 1
+```
+
+![screenshot](screenshots/metersPerUnit_1_usdrecord_22.08.png)
+_metersPerUnit_1.usda, usdrecord 22.08_
+
+## MetersPerUnit = 10
+
+[metersPerUnit_10.usda](./metersPerUnit_10.usda)
+
+```usda
+metersPerUnit = 10
+```
+
+![screenshot](screenshots/metersPerUnit_10_usdrecord_22.08.png)
+_metersPerUnit_10.usda, usdrecord 22.08_
+
+## MetersPerUnit mixed
+
+[metersPerUnit_mix.usda](./metersPerUnit_mixed.usda)
+
+This stage sets `metersPerUnit = 100` and also sublayers `metersPerUnit_1.usda` and `metersPerUnit_10.usda` with different values.
```

Comment:
Is this intended to demonstrate a typical usage of mixed MPU, and/or to re-emphasize that USD itself won't handle any auto-scaling to reconcile divergent MPU across layer stacks? Would be great to call that out in the documentation; thanks!

Reply:
I'll extend the documentation. The file is mainly aimed at showing that auto-scaling doesn't happen in the current implementation.